### ISSUE TITLE
Example 2 for G195 is too inaccurate

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#1305]
[G195](https://w3c.github.io/wcag/techniques/general/G195):

> A Web page includes a form inside a table. The borders of both the table and the form elements are thin, black lines. When focus lands on a form element, the element is outlined with a 5 pixel red line that is partially transparent. 

As long as "red" is not defined more precisely, it is unclear whether this color meets the contrast requirements, especially since part of the red is transparent (how large is this part?). The example should be removed or clarified to comply with SC 2.4.11 and 2.4.12
